src/recordlinker/schemas/pii.py

```diff
@@ -437,3 +448,70 @@ def blocking_values(self) -> typing.Iterator[tuple[models.BlockingKey, str]]:
             # a PII data dict could have multiple given names
             for val in self.blocking_keys(key):
                 yield key, val
+
+
+_STATE_NAME_TO_CODE = {
```

Comment:
What do you think about moving this to `src/assets/states.json`?  We have a [function in utils/path.py](https://github.com/CDCgov/RecordLinker/blob/main/src/recordlinker/utils/path.py#L17) to help with loading of the data.